default/python/AI/ColonisationAI.py

```diff
@@ -639,7 +641,7 @@ def get_base_outpost_defense_value():
     # for now, just combing these for rough composite factor
     # since outposts have no infrastructure (until late game at least), many of these have less weight
     # than for colonies
-    result = 3 * (0.1 + net_count) * (1 + regen_count/3.0) * (1 + garrison_count/6.0) * (1 + shield_count/3.0)
+    result = 3 * (0.1 + net_count) * (1 + regen_count / 3.0) * (1 + garrison_count / 6.0) * (1 + shield_count / 3.0)
```

Comment:
This is a case where I think the original code probably meets PEP8 *better* than the code proposed here does.

> If operators with different priorities are used, consider adding whitespace around the operators with the lowest priority(ies).
Yes:
x = x*2 - 1
No:
x = x * 2 - 1

But it is probably the " * " between the parentheses that is throwing off the tool.  CJ, pycodestyle would support a # noqa directive being added to this line, wouldn't it?  Do the Travis pep8 checks also support that?